client/src/components/Menu/Menu.scss

```
@@ -18,14 +18,12 @@
   @include media-breakpoint-down(sm) {
     position: absolute;
     top: 0;
-    left: -100%;
-    width: calc(100% - 59px) !important; // sass-lint:disable-line no-important
-    transition: left $transition-speed-fast ease-in;
-
+    left: -260px;
+    width: 250px;
+    box-shadow: 1px 0 10px rgba(0, 0, 0, 0.2);
+    transition: transform $transition-speed-fast;
     &--open {
-      left: 0;
-      height: 100% !important; // sass-lint:disable-line no-important
-      box-shadow: 1px 0 10px rgba(0, 0, 0, 0.2);
+      transform: translateX(260px) !important;
```

I can see there is the use of !important in the old version of the code but we try to avoid using it where possible. Can this be removed and achieved by increasing the specificity of your selector?

---

Yes, of course, this !important can be removed 
```suggestion
      transform: translateX(260px);
```

---

I did some updates, I think that now it's better.